### ISSUE TITLE
[iceprod2] add site to global pool (Trac #1484)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1476]
We need a way to add a site to the global pool.  This can be a manual script for now, taking the name of the site and returning a valid passkey that must be installed in the site config.

See attachment for one way to do this.

Migrated from https://code.icecube.wisc.edu/ticket/1484
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:28", 
    "description": "We need a way to add a site to the global pool.  This can be a manual script for now, taking the name of the site and returning a valid passkey that must be installed in the site config.\n\nSee attachment for one way to do this.", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067088921308", 
    "component": "iceprod", 
    "summary": "[iceprod2] add site to global pool", 
    "priority": "critical", 
    "keywords": "", 
    "time": "2015-12-11T20:05:13", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20151211T20:07:44: david.schultz changed attachment from "" to "add_site_to_pool.jpg"

**Comment by icecube-trac** [comment]
Trac update at 20160106T21:04:47: 
* david.schultz changed status from "new" to "closed"
* david.schultz commented:

> In changeset 140738:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="140738"
> add a site to the global pool using two helper scripts. change iceprod_server to default to daemon mode. closes #1476
> ```


* david.schultz changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:28: 
* olivas changed milestone from "IceProd 2.0" to ""
* olivas commented:

> Milestone IceProd 2.0 deleted
